packages/docs/docs/concepts/progressive-loading/volumeProgressive.md

```diff
@@ -0,0 +1,156 @@
+---
+id: volumeProgressive
+---
+
+# Volume Progressive Loading
+
+The volume progressive loading extends the basic stack loading with the ability
+to interleave various images, interpolating them from a reduced resolution
+version both intra and inter image. That is, individual images might be fetched
+initially at 1/4 size (256x256 for a CT), and then only the initially displayed
+image plus every 4th image, with other images being interpolated. In this case,
+replicate interpolation is used to minimize interpolation overhead. Finally,
+after the lossy initial versions are fetched, the remaining images are fetched.
+
+The default retrieve ordering is below, where Decimate is described as the
+interval between images included, and the offset in that set.
+
+- Initial images, full resolution
+- Decimate 4/3 partial resolution
+  - Interpolate images -2...+1 (nearest neighbors)
+- Decimate 4/1 partial resolution
+- Decimate 4/2 full resolution
+- Decimate 4/4 full resolution
+- Decimate 4/3 full resolution
+- Decimate 4/1 full resolution
+
+The same ordering is done if partial resolution is not configured, except that
+the last two stages are never run because the partial resolution has already
+loaded those. This DOES allow the interpolation of results to appear very quickly.
+
+# Types of Partial Resolution
+
+There are a few types of partial resolution image:
+
+- lossy images are original resolution/bit depth, but lossy encoded
+- thumbnail images are reduced resolution images
+- byte range images are a prefix of the full resolution, followed by
+  retrieving the remaining data. This only works for images like HTJ2K encoded
+  in resolution first ordering.
+
+# Creating Partial Resolution Images
+
+Static DICOMweb has been enhanced to add the ability to create partial resolution
```

Comment:
What would be the case when other PACS systems support HTJ2K? Are they going to add an option to retrieve all of lossy, thumbnail, byte range? Can we discuss this here as well?

I feel like we are writing this document around the static WADO toolkit, and we should absolutely avoid that mistake.

Reply:
Is the new HTJ2K standard (that you are part of), dictating about all of these scenarios so that later on we don't need to change our implementation for different vendors?

Reply:
The new standard is only dictating the transfer syntax for the HTJ2K that allows use of streaming and/or byte range request, so everything else is beyond the scope of that.